src/components/Menus/Sidebar/ProjectsPanel/ProjectsPanel.jsx

```diff
@@ -40,11 +40,10 @@ const ProjectsPanel = () => {
       heading={t("projectsPanel.projects")}
       Button={() =>
         isLoggedIn ? (
-          <Button
-            text={t("projectsPanel.yourProjectsButton")}
+          <DesignSystemButton
             className="btn--primary projects-panel__your-projects-button"
-            buttonIconPosition="right"
             href={`/${locale}/projects`}
+            text={t("projectsPanel.yourProjectsButton")}
           />
         ) : null
```

Comment:
🙏 any chance you could get rid of this redundant ternary too? I've been doing it as an when to remove all these `null`'s littering the place
```js
isLoggedIn && (
  ...
)
```